applovin_max/lib/src/ad_classes.dart

```diff
@@ -44,12 +46,16 @@ class MaxError {
   final int code;
   /// The error message for the error.
   final String message;
+  /// The failure info for ad load
+  final String adLoadFailureInfo;
```

Comment:
We should remove the API `adLoadFailureInfo` entirely, and just let the pub use the waterfall API